src/utils/sources-tree/utils.js

```diff
@@ -76,7 +76,7 @@ export function createParentMap(tree: Node): WeakMap<Node, Node> {
 }
 
 export function getRelativePath(path: string) {
-  const re = /(http(?:s?):\/\/(?:www\.)?[a-z0-9\-.]+)\/(.*)/i;
```

Comment:
can we borrow some good ideas from `getUrl` to find the relative path: https://github.com/devtools-html/debugger.html/blob/master/src/utils/sources-tree/getURL.js#L17-L38

I think `url.parse` will give us what we want. at a minimum we won't have to write a regex to handle the protocol/scheme thing